tools/harness-thci/OpenThread.py

```diff
@@ -3619,7 +3621,7 @@ def _connect(self):
             for _ in range(int(timeout / 0.5)):
                 time.sleep(0.5)
                 try:
-                    self.__handle = serial.Serial(self.port, 115200, timeout=0)
+                    self.__handle = serial.Serial(self.port, 115200, timeout=0, write_timeout=1)
```

Comment:
any background for this change?

Reply:
In case there is a problem with the device (e.g. it doesn't respond on serial) without write_timeout we may stuck and block GRL from discovering other devices. With write_timeout set the device can be skipped and the other devices can be still discovered

Reply:
Thansk for clarififation. I thought `write_timeout` is same as `timeout=0` by default.